packages/styles/scss/components/leadspace/_leadspace.scss

```
@@ -473,13 +563,48 @@ $btn-min-width: 26;
     padding-right: $spacing-05;
     width: 95%;
     z-index: 1;
-    @include type-style(fluid-heading-06, true);
 
-    /* stylelint-disable declaration-no-important */
+    ::slotted(span) {
+      color: $leadspace-heading-highlight;
+    }
+
     ::slotted(h1) {
-      all: unset !important; /* stylelint-disable-line declaration-property-value-blacklist */
+      display: none;
+    }
+  }
+
+  :host(#{$c4d-prefix}-leadspace-heading[type-style='fluid-heading-05']) {
+    @include type-style(fluid-heading-05, true);
+  }
+
+  :host(#{$c4d-prefix}-leadspace-heading[type-style='display-01']) {
+    @include type-style(display-01, true);
+  }
+
+  :host(#{$c4d-prefix}-leadspace)[size='super'] {
+    @include breakpoint(lg) {
+      .#{$prefix}--leadspace__overlay {
+        height: 640px;
```

should we convert this to rem?
```suggestion
        height: rem(640px);
```

---

oh yeah i have seen that! you might be missing the utility import

---

oh it should be this import `@use '@carbon/styles/scss/utilities' as *;`